### ISSUE TITLE
Getting error 'java/lang/NoClassDefFoundError': vowpalWabbit/responses/ActionProb

### ISSUE THREAD

**Issue by pallavi080596** [indeedeng/vowpal-wabbit-java#9]
Hi,

I am trying to run contextual MAB algo by creating this learner:
`static VWActionProbsLearner  learner = VWLearners.create("--cb_explore_adf -q UA  --softmax --lambda 20 --noconstant --permutations --learning_rate 0.8 --readable_model path.txt --invert_hash path_1.txt -f saved_model.vw --holdout_off --early_terminate 2");
`
After this, I am trying to predict using this command:
` ActionProbs[] testPreds = new ActionProbs[]{learner.predict(vwTextExample)};`

While calling this, I am getting error:
`Getting error 'java/lang/NoClassDefFoundError': vowpalWabbit/responses/ActionProb`

Can you please help here, what can be the issue ?

**Comment by pallavi080596** [comment]
``
